src/librbd/io/Dispatcher.h

```diff
@@ -86,6 +86,12 @@ class Dispatcher : public DispatchInterfaceT {
     on_finish->complete(0);
   }
 
+  bool dispatch_exist(DispatchLayer dispatch_layer) override {
```

Comment:
Nit: see previous comment re: removing this method

Reply:
Removed